packages/@react-spectrum/s2/src/Field.tsx

```diff
@@ -80,9 +81,10 @@ function FieldLabel(props: FieldLabelProps, ref: DOMRef<HTMLLabelElement>) {
         contain: {
           labelPosition: {
             top: 'inline-size'
-          }
+          },
+          isQuiet: 'none'
```

Comment:
adding this allows the label be on a single line regardless of the picker's selected text

Reply:
I guess we'll have to figure out if we want this, might interfere with form layouts

Reply:
yeah im open to either. for tabs, there isn't going to be a label anyway so this won't matter too much. i added this to mainly to match the v3 behavior. i don't really think isQuiet should be used in forms given that the other form components don't have isQuiet styles but if you're curious, this is what it looks like having an isQuiet Picker in a form
<img width="275" alt="Screenshot 2024-08-06 at 11 03 17 AM" src="https://github.com/user-attachments/assets/16857cf6-838f-4bfe-a1a2-749cff4ca8a8">
<img width="267" alt="Screenshot 2024-08-06 at 11 03 25 AM" src="https://github.com/user-attachments/assets/7e3d1fa7-3e69-48b9-a118-1a9beced1394">

for the second image, i had to apply a width to the form, otherwise, the text would overflow